elixir/apps/web/lib/web/live/resources/show.ex

```diff
@@ -116,11 +125,16 @@ defmodule Web.Resources.Show do
         Danger zone
       </:title>
       <:actions>
-        <.delete_button>
+        <.delete_button phx-click="delete" phx-value-id={@resource.id}>
```

Comment:
@bmanifold please add a data-confirm attr: https://github.com/firezone/firezone/blob/main/elixir/apps/web/lib/web/live/actors/show.ex#L192

Reply:
🙏 Thank you!  I had not come across this attribute yet, but I was planning on trying to figure out a way to add a confirmation in at a later date.  I'll get this updated.